test/common/common/log_macros_test.cc

```diff
@@ -37,4 +37,24 @@ TEST(Logger, All) {
   // Misc logging with no facility.
   ENVOY_LOG_MISC(info, "fake message");
 }
+
+TEST(Logger, evaluateParams) {
+  auto i = 1;
```

Comment:
Please give this a type: uint32_t i = 1;

Reply:
OK. Will change that to uint32_t. Thanks.